GOVERNANCE.md

```diff
@@ -0,0 +1,29 @@
+# Governance Policy
+
+This document provides the governance policy for the SpotBugs Project.
+
+## SpotBugs Core Term
+
+The SpotBugs Core Team maintain the spotbugs/spotbugs GitHub repository.
+
+The GitHub team for SpotBugs Team is `@spotbugs/core-devs`. Members of the SpotBugs Core Team have write access to the repository.
+
+## How we propose changes
+
+Both SpotBugs Core Team and SpotBugs user can propose changes to the SpotBugs project via GitHub pull requests. Refer to `.github/CONTRIBUTING.md` to know more detailed requirements for your proposal.
+
+Once a pull request got two approvals from members of SpotBugs Core Team, pull request will be merged and shipped in the next release. Approver should not be the same to authors of the change.
```

Comment:
"got" -> "gets"

"the same to" -> "the same as the"

"authors of" -> "author(s) of"

Perhaps also add something about how to handle pull requests where all or most of the team has contributed as authors?

Reply:
> Perhaps also add something about how to handle pull requests where all or most of the team has contributed as authors?

Any idea do you have? How about asking project lead (iloveeclipse) to make a judge?

Reply:
One option: it's okay to have approvers who are also authors if there are at least three approvals (after the most recent changes). So everyone's work has been reviewed by at least two people who didn't write it, even if they wrote other parts.